Exiled.Events/Config.cs

```diff
@@ -103,5 +103,11 @@ public sealed class Config : IConfig
         /// </summary>
         [Description("Whether or not to log RA commands.")]
         public bool LogRaCommands { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the SCP-956 should be a capibara.
+        /// </summary>
+        [Description("Whether the SCP-956 should be a capibara.")]
+        public bool Is956Capybara { get; set; } = false;
```

Comment:
Sorry i can't accept that on Exiled config xD Even if i really like it it's would need to be separated plugin lol

Reply:
That's sad 😢